plugins/mcreator-localization/lang/texts.properties

```diff
@@ -90,11 +90,13 @@ dialog.animation_maker.template=Animation template:
 dialog.animation_maker.color=Animation color:
 dialog.animation_maker.saturation_lightness_lock=Lock saturation and lightness:
 dialog.animation_maker.preview=Preview
-dialog.animation_maker.add_frames_from_template=Add frames from template
```

Comment:
This key does not exist in master either so I am a bit confused where these do come from. Actually no key under "dialog.animation_maker." is in master, but GitHub indicates they are there.